packages/eslint-plugin-react-hooks/src/ExhaustiveDeps.js

```diff
@@ -179,6 +179,10 @@ export default {
       //               ^^^ true for this reference
       // const [state, dispatch] = useReducer() / React.useReducer()
       //               ^^^ true for this reference
+      // const [state, dispatch] = useReducer() / React.useActionState()
```

Comment:
Typo? I think you meant - `useActionState() / React.useActionState()`

Reply:
Thanks, fixed.